server/utils/writeResponse.ts

```diff
@@ -0,0 +1,18 @@
+import { Response } from "express";
+
+interface ResponseData {
+  statusCode?: number;
+  status?: string;
```

Comment:
What is `status`? If its replacement for `messageKey` it's not good. We need to return the message, but it should be the message key for translate it on FE.
This name is not enough